kodilla-collections/src/main/java/com/kodilla/collections/arrays/homework/CarsApplication.java

```diff
@@ -0,0 +1,40 @@
+package com.kodilla.collections.arrays.homework;
+
+import com.kodilla.collections.interfaces.homework.Audi;
+import com.kodilla.collections.interfaces.homework.BMW;
+import com.kodilla.collections.interfaces.homework.Car;
+import com.kodilla.collections.interfaces.homework.Mercedes;
+
+import java.util.Random;
+
+public class CarsApplication {
+
+    public static void main(String[] args) {
+
+        Random random = new Random();
+        Car[] cars = new Car[random.nextInt(15) + 1];
+        for (int n = 0; n < cars.length; n++)
+            cars[n] = drawCar();
+        for (Car car : cars)
+            CarUtils.describeCar(car);
+    }
+
+    public static Car drawCar() {
+
+        Random random = new Random();
+        int drawnCars = random.nextInt(3);
+        int increaseSpeed = getRandomIncreaseSpeed();
+        if (drawnCars == 1)
+            return new Mercedes(increaseSpeed);
+        else if (increaseSpeed == 2)
+            return new BMW(increaseSpeed);
+        else {
+            return new Audi(increaseSpeed);
+        }
+        return drawCar();
+    }
+
+    private static int getRandomIncreaseSpeed() {
+        return random.nextInt(250);
```

Comment:
Skad ta metoda ma wiedziec, co to jest random? 
najlepiej zadeklarowac pole statyczne w klasie typu Random